views/display_word.erb

```diff
@@ -0,0 +1,13 @@
+<% if @word_score[:score].class == Fixnum %>
+  <%= "#{ @word_score[:word].capitalize } is worth #{ @word_score[:score] } points." %>
+  <ul><% @word_score[:letters].each do |letter| %>
+    <% @letter = letter %>
```

Comment:
I might rename the variable `@letter` because this variable is actually storing a mapping between the letter and the score. Then it will be more clear in the display_letter view because in that location there is no insight into how this variable was set and what kind of data is in it.